airflow/www/views.py

```diff
@@ -2288,6 +2292,53 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_repo = git_hash = None
+        try:
+            p = subprocess.Popen('pip freeze | grep airflow',
+                                 shell=True,
+                                 stdout=subprocess.PIPE,
+                                 stderr=subprocess.STDOUT)
+            for line in p.stdout.readlines():
+                parts = line.split("@")
+                # parts = ['-e git+git', 'github.com:apache/incubator-airflow.git',
+                #          '72ab63e83d68c09a77ede120ea316f5c1b9ff4d0#egg=airflow\n']
+                git_repo = parts[1]
+                git_hash_and_egg_info = parts[2]
+                sub_parts = git_hash_and_egg_info.split("#")
+                # sub_parts =  ['72ab63e83d68c09a77ede120ea316f5c1b9ff4d0', 'egg=airflow\n']
```

Comment:
Ah ok, maybe put the comment above the assignment and replace "sub_parts = " with "e.g." or maybe I just get confused easily and ignore me :).